tide-log/src/lib.rs

```diff
@@ -1,5 +1,7 @@
+#![cfg_attr(feature = "nightly", deny(missing_docs))]
+#![cfg_attr(test, deny(warnings))]
```

Comment:
This will still stop developers from being able to run tests locally when their code has warnings in it. I think having the warnings denied when running clippy in CI is enough: https://github.com/rustasync/tide/blob/2878abad4695a9fa85a90818c31eb844363bce35/.travis.yml#L21

Reply:
Oh, yikes, yeah, this makes no sense. Mechanically copied over the entire header part from tide. Fixing it up